Description: Accomodate non-standard build
 dieharder (3.31.1-8) unstable; urgency=medium
 .
   * debian/control: Change Build-Depends: from texlive-generic-recommended
     to texlive-plain-generic 				(Closes: #941516)
 .
   * debian/control: Set Build-Depends: to 'debhelper (>= 10)'
   * debian/control: Set Standards-Version: to current version
   * debian/control: Add Vcs-Browser: and Vcs-Git:
   * debian/compat: Increase level to 9
Author: Dirk Eddelbuettel <edd@debian.org>
Bug-Debian: https://bugs.debian.org/941516

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2019-10-03

--- dieharder-3.31.1.orig/include/dieharder/libdieharder.h
+++ dieharder-3.31.1/include/dieharder/libdieharder.h
@@ -14,6 +14,7 @@
 #include <stdarg.h>
 #include <string.h>
 #include <sys/time.h>
+#include <stdint.h>
 
 /* This turns on uint macro in c99 */
 #define __USE_MISC 1
